GPT/gpt.talon

```diff
@@ -7,6 +7,12 @@ model ask <user.text>$:
     result = user.gpt_answer_question(text)
     user.paste(result)
 
+# Say your prompt directly and the AI will apply it to the selected text
+model go <user.text>$:
```

Comment:
do we want to support static prompts as well?

Reply:
This is the primary way that I use this, I do want to allow this.

Reply:
I misunderstood at first. Maybe? Good question.

Reply:
Not sure if it makes sense to have static prompts unless you want to make function calling less explicit. Since if a user wants to apply a static prompt they can just say the name.

Reply:
I think it does make sense to have static prompts because the function calling is not actually different except in the level of trust you are giving to the LLM.  This is why I originally didn't have this as a separate Python file-there isn't a different use case. It's an implementation detail.